.github/workflows/dsf_poc_standalone.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     uses: imperva/dsfkit/.github/workflows/dsf_poc_cli.yml@master
     with:
       use_modules_from_terraform_registry: true
-      explicit_ref: master
+      explicit_ref: ${{ github.ref_name }}
```

Comment:
I saw that all other places use "github.ref" and not "github.ref_name".
Is it intended? are they different?